server/manager/src/main/java/org/apache/accumulo/manager/compaction/queue/CompactionJobPriorityQueue.java

```diff
@@ -107,7 +108,7 @@ public boolean equals(Object o) {
   // efficiently removing entries from anywhere in the queue. Efficient removal is needed for the
   // case where tablets decided to issues different compaction jobs than what is currently queued.
   private final TreeMap<CjpqKey,CompactionJobQueues.MetaJob> jobQueue;
-  private final int maxSize;
+  private Supplier<Integer> maxSize;
```

Comment:
The way this is used seems like it could be an AtomicInteger (or maybe interger if only accessed in sync method) instead of supplier.  The supplier passed to constructor seems to contain a fixed value or final constant and the set method replaces the supplier.

Reply:
Changed this to an AtomicInteger in 18d2243